src/Ontology/Oep4.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright © 2017-2020 Trust Wallet.
```

Comment:
Update copyright for new added files
```suggestion
// Copyright © 2017-2022 Trust Wallet.
```

Reply:
done